redi-connect-front/src/components/molecules/PipeList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { Element } from 'react-bulma-components'
+import './PipeList.scss'
+
+interface Props {
+  items: string[]
+}
+
+const PipList = ({ items }: Props) => {
+  return (
+    <Element renderAs="ul" className="pipe-list">
+      {items.map((language: any, index: number) => <Element renderAs="li" key={`${language}_${index}`}>{language}</Element>)}
```

Comment:
You can safely simplify and improve this down to:
```
{items.map(language => <Element renderAs="li" key={language}>{language}</Element>)}
```

Recommended reading: https://reactjs.org/docs/lists-and-keys.html

TL;DR: if you can avoid it, don't use array index in the element's key